xarray/tests/test_backends.py

```diff
@@ -873,6 +886,7 @@ def create_tmp_files(nfiles, suffix='.nc', allow_cleanup_failure=False):
 
 @requires_netCDF4
 class BaseNetCDF4Test(CFEncodedDataTest):
+    """Tests for both netCDF4-python and h5netcdf."""
 
```

Comment:
this comment contradicts the ``@requires_netCDF4`` decorator immediately above

Reply:
removed